lib/associations/belongs-to-many.js

```diff
@@ -287,6 +287,16 @@ class BelongsToMany extends Association {
        */
       removeMultiple: 'remove' + plural,
       /**
+       * Un-associate all instances.
+       *
+       * @param {Object} [options] Options passed to `through.destroy`
```

Comment:
Hmm the `destroyAssociation` accept the same parameters? I understand that you are saying that `removeMultiple` should destroy all associations if no array is passed.